### ISSUE TITLE
'High Contrast Black' theme Asset Upload WCAG AAA contrast compliance

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1001]
**Describe the bug**
The 'Asset Upload' component color contrast ratio is not 7:1 or above.

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
React

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/asset-upload.html
2. Check the 'Upload Link' text and icon color contrast with the background

**Expected behavior**
If we want WCAG AAA compliance, the 'Upload Link' text and icon color contrast ratio should be 7:1 or above.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81739267-e1369780-944f-11ea-9241-fbfa97e93f06.png)
![image](https://user-images.githubusercontent.com/43764832/81739353-0cb98200-9450-11ea-8632-af7eb65075d9.png)
![image](https://user-images.githubusercontent.com/43764832/81739389-222eac00-9450-11ea-9224-2cf9d75fee64.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
This  component is deprecated and has been replaced by File Uploader.
